### PR TITLE
*: clean old owner related codes

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve? 
#2574 

### What is changed and how it works?
None

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)

Code changes

 - Has exported variable/fields change

Side effects

 - Breaking backward compatibility

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by amyangfei — APPROVED** [review]

File: cdc/server_test.go [hunk 1]
```diff
@@ -19,6 +19,9 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/pingcap/ticdc/cdc/kv"
+	"go.etcd.io/etcd/clientv3"
+
 	"github.com/pingcap/check"
 	"github.com/pingcap/ticdc/pkg/config"
 	"github.com/pingcap/ticdc/pkg/etcd"
```
Comment (amyangfei) [inline cdc/server_test.go:27]: Better to remove the blank line and group the import

Comment (asddongmen) [reply cdc/server_test.go]: done
thx ~!

File: tests/cyclic_abc/run.sh [hunk 2]
```diff
@@ -99,23 +99,23 @@ function run() {
     start_ts=$(run_cdc_cli tso query --pd=http://$UP_PD_HOST_1:$UP_PD_PORT_1)
 
     run_cdc_cli changefeed create --start-ts=$start_ts \
-        --sink-uri="mysql://root@${DOWN_TIDB_HOST}:${DOWN_TIDB_PORT}/?safe-mode=false" \
+        --sink-uri="mysql://root@${DOWN_TIDB_HOST}:${DOWN_TIDB_PORT}/?safe-mode=true" \
```
Comment (amyangfei) [inline tests/cyclic_abc/run.sh:102]: This change should be picked to release branch

**Review by amyangfei — COMMENTED** [review]
By the way, some outdated tests should also be removed in release branches, is there any plan for it

**Comment by asddongmen** [comment]
> By the way, some outdated tests should also be removed in release branches, is there any plan for it

ok，I will open an issue and do the jobs.

**Review by 3AceShowHand — APPROVED** [review]

**Pull request merged**